airflow-core/src/airflow/ui/src/layouts/Nav/PluginMenuItem.tsx

```diff
@@ -23,19 +23,31 @@ import { RiArchiveStackLine } from "react-icons/ri";
 import { Link as RouterLink } from "react-router-dom";
 
 import type { ExternalViewResponse } from "openapi/requests/types.gen";
+import { useColorMode } from "src/context/colorMode";
 import type { NavItemResponse } from "src/utils/types";
 
 import { NavButton } from "./NavButton";
 
 type Props = { readonly topLevel?: boolean } & NavItemResponse;
 
-export const PluginMenuItem = ({ icon, name, topLevel = false, url_route: urlRoute, ...rest }: Props) => {
+export const PluginMenuItem = ({
+  icon,
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  icon_dark_mode,
```

Comment:
I think this might make eslint happy

```suggestion
  icon_dark_mode: iconDarkMode,
```

Reply:
Good hint, did not know about this trick! This made it happy! Thanks!